saml.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/* Copyright (c) 1998-2017 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * @author Michael Jansen <mjansen@databay.de>
+ */
+
+require_once 'libs/composer/vendor/autoload.php';
+$as = new SimpleSAML_Auth_Simple('default-sp');
```

Comment:
I think i get your point. I remember some troubles with standard session handler, ended up storing session data in sql.